### PR TITLE
Change some options for `prevent-abbreviations` rule

### PR DESCRIPTION
**Pull request by fisker**
The PR will make this code bellow report an error by default

```js
const getDocsUrl = require('./utils/get-docs-url');
```

without breaking https://github.com/sindresorhus/eslint-plugin-unicorn/pull/237#issuecomment-470942156

```js
import tempWrite from 'temp-write';
```

and

```js
import tempWrite from './node_modules/temp-write';
```

moduleId starts with `.`  `/` ~`@/`~ and without `node_modules` is considered `internal-module`.

The `internal` naming and `internal-module` logic may need discuss.

//cc @futpib @sindresorhus

### CODE REVIEW

**Comment by futpib** [comment]
What is the meaning of a module name starting with `@/`?

**Comment by fisker** [comment]
@ is webpack alias, means project root

**Comment by fisker** [comment]
reminds me there is also a `~`, or  maybe we do  external check, any thoughts?

**Comment by futpib** [comment]
Webpack aliases can be absolutely arbitrary and using `@` as an alias for the root or the project is not something I've seen used. I don't think we should support something that is not even a widespread convention (and definitely not a standard).

Anyway, what is the issue you are trying to solve with this PR?

**Comment by fisker** [comment]
I want imports report error name is using abbreviation,  but https://github.com/sindresorhus/eslint-plugin-unicorn/pull/237#issuecomment-470942156 don't want check npm packages import, so I made this PR to only target `internal-module`s resolve things like example I wrote in the comment above , and this https://github.com/sindresorhus/eslint-plugin-unicorn/blob/dbf4b7996a1aba6deef51916a5435092c130b34e/test/get-documentation-url.js#L2 It's not reported because it's `import`  .

**Comment by fisker** [comment]
@futpib @sindresorhus Are you ok with this change?

**Comment by sindresorhus** [comment]
> Webpack aliases can be absolutely arbitrary and @ as an alias for the root of the project is not something I've seen used. I don't think we should support something that is not even a widespread convention (and definitely not a standard).

👍

File: docs/rules/prevent-abbreviations.md [hunk 1]
```diff
@@ -156,12 +156,28 @@ Pass `"extendDefaultWhitelist": false` to override the default `whitelist` compl
...
+```js
+import * as err from './err';
+```
+
+```js
+import err from '/err';
+```
+
+```js
+const err = require('@/err');
```
Comment (sindresorhus) [inline docs/rules/prevent-abbreviations.md]: I don't think we should support `@/`.

File: docs/rules/prevent-abbreviations.md [hunk 2]
```diff
@@ -177,12 +193,20 @@ const err = require('err');
...
-Pass `"checkShorthandImports": true` to check variables declared in shorthand import.
+- `'internal'` - Check variables declared in shorthand import, **but only for internal modules**.
+- `true` - Check variables declared in shorthand import.
+- `false` - Don't check variables declared in default shorthand import.
 
-With this set to `true` the following code will be reported.
+By default, the following code will be reported:
+
+```js
+import {prop} from './ramda';
```
Comment (sindresorhus) [inline docs/rules/prevent-abbreviations.md]:
```suggestion
import {prop} from './foo';
```

It's confusing to use a word that is a known package name.

**Comment by sindresorhus** [comment]
> Are you ok with this change?

Yes

**Pull request merged**